requirements.txt

```diff
@@ -1,9 +1,19 @@
+# for database
+-r db/requirements.txt
+
+# for onboarding
+-r onboarding/requirements.txt
+
+# for Rasa agent
 -r Rasa_Bot/requirements.txt
-alembic
-connexion
+
+# for rasa-scheduler
+-r requirements-scheduler.txt
```

Comment:
This is the odd one out. I think this PR is a good opportunity to move these requirements from `requirements-scheduler.txt` to `rasa_scheduler/requirements.txt`.